### ISSUE TITLE
deleteleaf method bug

### ISSUE THREAD

**Issue by zyzvnnb** [crsl4/PhyloNetworks.jl#153]
I am using `deleteleaf` method. When I try to delete "A13", it seems like "Pae102" also be deleted from the network topology (although "Pae102" is still in tip labels of the network object)
Below is the code to demonstrate this issue, thank you for checking this!

![image](https://user-images.githubusercontent.com/56092017/101760109-646ab980-3b15-11eb-97be-267616d7b676.png)
![image](https://user-images.githubusercontent.com/56092017/101760194-87956900-3b15-11eb-8235-db69b03f00a3.png)
![image](https://user-images.githubusercontent.com/56092017/101760783-48b3e300-3b16-11eb-8c8f-dad956fd5245.png)

**Comment by crsl4** [comment]
Thanks @zyzvnnb ! Do you mind writing the code as text that we can copy and reproduce? Thanks!

**Comment by cecileane** [comment]
Which version of Julia and which version of PhyloNetworks are you using? You can do this to know:
```julia
versioninfo() # for julia version
using Pkg
Pkg.status() # for PhyloNetworks version (and other packages)
```

I suspect that the problem would go away if you use PhyloNetworks v0.12 (which fixes a related [issue](https://github.com/crsl4/PhyloNetworks.jl/issues/84)). You can refer to the documentation to see how the options for `deleteleaf!` have been expanded, from [v0.11](http://crsl4.github.io/PhyloNetworks.jl/v0.11/lib/public/#PhyloNetworks.deleteleaf!) to [v0.12](http://crsl4.github.io/PhyloNetworks.jl/v0.12/lib/public/#PhyloNetworks.deleteleaf!)

**Comment by cecileane** [comment]
please re-open if the issue persists. I'll assume it's been fixed in v0.12.
